### PR TITLE
Fix Docs for Providers

### PR DESCRIPTION
**Pull request by kaxil**
Updates the link from 'latest' to 'stable' (Just updates `docs-archive/apache-airflow-providers`)

### CODE REVIEW

**Comment by potiuk** [comment]
Hmmm. Those are a bit more than latest -> stable.. Are those changes right ?

**Comment by kaxil** [comment]
> Hmmm. Those are a bit more than latest -> stable.. Are those changes right ?

Yup, auto-generated. Fixes link like twitter.com to twitter.com/ApacheAirflow same for Github and other links

**Comment by potiuk** [comment]
Good for me :)

**Review by potiuk — APPROVED** [review]

File: docs-archive/apache-airflow-providers/_modules/index.html [hunk 1]
```diff
@@ -29,8 +29,8 @@
     
 
 <header>
-
-    <nav class="js-navbar-scroll navbar">
+    <a href="https://airflowsummit.org" target="_blank" class="d-block fixed-top px-3 py-2 bg-success text-center text-bold text-white">Airflow Summit 2021 is coming July 8-16. Register now!</a>
```
Comment (potiuk) [inline docs-archive/apache-airflow-providers/_modules/index.html:32]: Maybe we should remove that one:)

Comment (kaxil) [reply docs-archive/apache-airflow-providers/_modules/index.html:32]: Oh yeah let me remove that on entire website in one-go

**Pull request merged**